microsoft-edge/devtools-guide-chromium/evaluate-performance/selector-stats.md

```diff
@@ -21,40 +21,86 @@ In your performance recordings, some of these long-running tasks may be **Recalc
 
 When you have long-running **Recalculate Style** events, you can use the **Selector Stats** feature to understand which of your CSS selectors are taking up the most time and resulting in slow performance.
 
-## How to use Selector Stats
+The **Selector Stats** feature provides statistics about the CSS rule selectors that were involved in one or more **Recalculate Style** events within a performance recording.
 
-To see **Selector Stats** during **Recalculate Style** events:
+## View CSS rule selector statistics
+
+#### Record a performance trace with Selector Stats enabled
+
+To view the statistics of your CSS rule selectors during long-running **Recalculate Style** events, first record a performance trace with the Selector Stats feature enabled.
+
+The Selector Stats feature isn't always enabled because it adds more overhead to your performance recordings. You should only leave it turned on when you need to investigate the performance of **Recalculate Style** events  and other rendering information:
```

Comment:
```suggestion
The Selector Stats feature isn't always enabled, because it adds more overhead to your performance recordings. You should only leave it turned on when you need to investigate the performance of **Recalculate Style** events and other rendering information:
```